### DISCUSSION TITLE
Wio-E5 STM32WLE5JC Module | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1054] [Q&A]
# Wio-E5 STM32WLE5JC Module | Seeed Studio Wiki

Get One Now 🖱️

https://wiki.seeedstudio.com/LoRa-E5_STM32WLE5JC_Module/

**Comment by wdavison** [comment]
Is it possible to send messages between two W5s connected to TTN with your pre-configured
 AT command set?  I know this can be done in TEST mode in a point to point senerio, but wondering about through a server like TTN. I am actually using local chirpstack and can send to a MQTT topic.
